src/gui/org/deidentifier/arx/gui/view/impl/menu/DialogAnonymization.java

```diff
@@ -89,8 +94,17 @@ public DialogAnonymization(Shell shell, Model model) {
         this.configuration = model.getAnonymizationConfiguration();
         this.configurationValid = true;
         
-        // Determine if optimal search is available
-        this.optimalSearchAvailable = model.getSolutionSpaceSize() <= model.getHeuristicSearchThreshold();
+        // Determine available search options
+        EDDifferentialPrivacy edp = model.getInputConfig().getCriterion(EDDifferentialPrivacy.class);
+        if (edp != null) {
+            this.optimalSearchAvailable = false;
+            this.heuristicSearchStepLimitAvailable = edp.isDataDependent();
+            this.heuristicSearchTimeLimitAvailable = false;

```

Comment:
PrivacyModel.isHeuristicSearchSupported(); defaults to true
PrivacyModel.isHeuristicSearchWithTimeLimitSupported(); defaults to true
-> Overwrite for EDP